ONSdigital/rm-case-service
# Generate new access code

# Motivation and Context
To generate a new access (IAC) code

# What has changed
Dropped case iac code column after migrating existing values from caseiacaudit table. IAC codes should only be stored/read (using latest) from this table.

# How to test?
test in conjunction with [rasrm-acceptance-tests PR](rasrm-acceptance-tests](https://github.com/ONSdigital/rasrm-acceptance-tests/pull/168)

delete all caseiacaudit records

start casesvc service on this branch, caseiacaudit table should be re-populated with iac values from case table and column now removed

"Generate new enrolment code" function from response-operations-ui should add a record to caseiacaudit table (only if no active iac code)

# Links
https://trello.com/c/WTjuaOv5/190-sus004-generate-a-new-access-code-iac-for-an-lms-case
